src/env.rs

```diff
@@ -64,6 +65,10 @@ fn default_log_level() -> String {
     "WARN".to_string()
 }
 
+fn default_database_url() -> String {
```

Comment:
We don't need a default, if its not there it can panic on startup

Reply:
@HarryET this change served for `tests`, have you ever run it on my PR? Do you want me provide a dummy string instead? If i don't set it, tests will fail.

Reply:
Yeah I know tests will fail, just update the tests